controllers/login.controller.es6

```diff
@@ -56,7 +113,7 @@ export default class LoginController {
       let alert = new Alert({
         title: 'Invalid secret key',
         text: 'Secret keys are uppercase and begin with the letter "S."',
-        type: Alert.TYPES.ERROR
+        type: Alert.TYPES
```

Comment:
Removed by mistake?

Reply:
Yes, trying to make my IDE give me hints about alert types